src/validation.cpp

```diff
@@ -3849,26 +3849,30 @@ bool LoadChainTip(const CChainParams& chainparams)
         // that we always have a chainActive.Tip() when we return.
         LogPrintf("%s: Connecting genesis block...\n", __func__);
         CValidationState state;
-        if (!ActivateBestChain(state, chainparams)) {
+        if (!ActivateBestChain(state, chainparams, std::shared_ptr<const CBlock>())) {
```

Comment:
In commit "Move LoadChainTip into CChainState",

Use `MakeShared()`?

Reply:
shared_ptr() creates a pointer-to-null, not a default-constructed CBlock.

Reply:
Any particular reason to pass it explicitly?

Reply:
Indeed, this change could be reverted (it's the default value).